### DISCUSSION TITLE
REST API request detailed use of App Service authentication token

### DISCUSSION THREAD

**Discussion by scastellanosnqn** [polymind-inc/acmebot#605] [Q&A] [answered]
Hello, I'm trying to implement to invoke REST API endpoints from bash using cURL.
I've manage to make it work but disabling 'App Service authentication' and using the host key as authorization, but can't do it keeping 'App Service authentication' enabled so that we can do both, the REST API and the dashboard, which I found really handy.

Could someone using the REST API elaborate a bit about getting the Identity Provider token an using it in an example request ?

Thanks in advance.

**Comment by shibayan** [comment, marked as answer]
Published instructions for calling the API from external applications even when Azure AD authentication is enabled.
https://github.com/shibayan/keyvault-acmebot/wiki/Authentication-Methods-for-REST-API
